### DISCUSSION TITLE
🔊 Announcements

### DISCUSSION THREAD

**Discussion by JohanCSmit** [COS301-SE-2022/Exploring-Self-Sovereign-Identity#7] [Announcements]
Hi everyone, please try communicating as much as possible through these discussions.
I'm excited to work with you all and let's put in all our effort!

**Comment by JohanCSmit** [comment]
Please have a look at our GitHub and provide some feedback/suggestions! @10416260 @RebeccaPillay @MxolisiMkanzi @H-e-n-r-i

**Reply by RebeccaPillay** [reply]
Looking great so far! Will add information where necessary and let you know about any suggestions when I have them.

**Comment by JohanCSmit** [comment]
Remember to start hitting the weekly commit and issue-closing targets. Please make use of GitHub for communication and let's work hard this week!
@RebeccaPillay @10416260 @H-e-n-r-i @MxolisiMkanzi
